pages/glossary/visual-context.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Visual context
+key: visual-context
+---
+
+The visual context of some piece of information (_e.g._ an element) is everything that is visually or logically located near it. Only [visible](#visible) information may be part of the visual context.
```

Comment:
I am inclined to ask for a more precise definition of "near". I think this is an incredibly difficult definition to write and applaud your ambition.

Reply:
That is indeed one of the main problem with that definition and the reason of the "distance" note. The "near" that need to be used depends a lot on why the user needs the definition in their current use case (the "context" in which the definition is used, but I'd like to avoid using this term here to keep things not too complicated).

If this definition is used to differentiate otherwise similar links in a table, then "near" will be very near, typically only the cell in which the link resides. If used to check whether such a link is descriptive, then "near" can be further away and include the table headers or caption, and even the headings above it. Possibly far away from it…

Reply:
I added some note in "visual context" to give a better (but still blurry) definition of "near".